controllers/grades.go

```diff
@@ -0,0 +1,50 @@
+package controllers
+
+import (
+	"context"
+	"github.com/BetterGR/grades-microservice/protos"
+	"google.golang.org/grpc/credentials/insecure"
+	"log"
+	"net/http"
+	"time"
+
+	"github.com/gin-gonic/gin"
+	"google.golang.org/grpc"
+)
+
+var grpcClient protos.GradesServiceClient
+
+func InitGRPCClient() {
+	// Initialize the gRPC client connection.
+	conn, err := grpc.NewClient("localhost:50051", grpc.WithTransportCredentials(insecure.NewCredentials()))
+	if err != nil {
+		log.Fatalf("Failed to connect to Grades Microservice: %v", err)
```

Comment:
No non-main function should call `log.Fatal`. When erring, functions should return an error.

Death from errors should occur in main, when necessary.